ports/espressif/esp-idf-config/sdkconfig.defaults

```diff
@@ -37,9 +37,11 @@ CONFIG_ESP_MAIN_TASK_STACK_SIZE=16384
 
 #
 # Wi-Fi
-#
-CONFIG_ESP_WIFI_DYNAMIC_RX_BUFFER_NUM=8
-CONFIG_ESP_WIFI_STATIC_TX_BUFFER=y
+
+# Broken for C6
+# CONFIG_ESP_WIFI_DYNAMIC_RX_BUFFER_NUM=8
+# CONFIG_ESP_WIFI_STATIC_TX_BUFFER=y
```

Comment:
Instead of commenting these out, just remove them. That'll inherit the default settings instead.

Reply:
Done!